py/h2o_wave/server.py

```diff
@@ -73,6 +73,21 @@ def __init__(self, username: str, subject: str, access_token: str, refresh_token
         """The access token of the user."""
         self.refresh_token = refresh_token
         """The refresh token of the user."""
+        self._session_id = session_id
+        """Session identifier. Do not access, internal use only."""
+    
+    async def force_token_refresh(self):
```

Comment:
Naming: IMHO this does necessarily not _forces_ the refresh of the token. I cannot confirm what implementation of the `TokenSource` is used server-side but default implementation IMHO refreshes the token only if it's expired. Prefixing it with `force_` might implied functionality that is not effectively implemented.

Ideas:
 - `ensure_fresh_token`: reflects what it really does on the back end
 - `token_provider`: see the plea comment

Reply:
I agree with `force_` prefix not being the best. I would go with `ensure_fresh_token` as it seems to be most flexible for both using it as explicit refresh and as a token provider, wdyt?

Reply:
Agree. Thanks.